lib/active_admin/csv_builder.rb

```diff
@@ -122,6 +115,15 @@ def humanize_name(name, resource, humanize_name_option)
 
     private
 
+    def add_collection_to_csv(csv, controller, csv_options)
+      (1..paginated_collection.total_pages).each do |page|
+        paginated_collection(page).each do |resource|
+          resource = controller.send :apply_decorator, resource
+          csv << CSV.generate_line(build_row(resource, columns, options), csv_options)
```

Comment:
Ugh. Looks like you have to pass in the local options from build instead of using self.options. I'm OK with leaving as a lambda, the options handling needs a deeper refactoring.

Reply:
@varyonic no problem, I updated it to be a lambda again!